src/SkeletalTrapezoidation.cpp

```diff
@@ -1519,9 +1532,17 @@ std::pair<SkeletalTrapezoidation::edge_t*, SkeletalTrapezoidation::edge_t*> Skel
 std::pair<Point, Point> SkeletalTrapezoidation::getSource(const edge_t& edge)
 {
     const edge_t* from_edge;
-    for (from_edge = &edge; from_edge->prev; from_edge = from_edge->prev) {}
+    for (from_edge = &edge; from_edge->prev; from_edge = from_edge->prev) 
+    {
+        
```

Comment:
If you're giving empty for loops that much space, _I'd say put a comment in it_ ... `{}` seems a lot more intentional, even if it's maybe against style.